pages/design/rule-design.md

```diff
@@ -39,34 +77,64 @@ A test aspect is a part of a test subject that must be available in order to pro
 
 - It is assumed that the following technologies are accessibility supported: HTML, CSS, WAI-ARIA, ... (See also WCAG-ACT-RULES-CG's [explanation on Accessibility Support](accessibility-support.html)).
 
-## Applicability
+For more details, see [ACT Rules Format: Assumptions](https://www.w3.org/TR/act-rules-format/#assumptions).
 
-Applicability describes which (elements of) web pages should be tested using the rule. These elements are known as test targets. Applicability must be written in plain language, so that it can be used by QA testers. Applicability must rely on well defined properties of the technologies that are tested. For example, a rule may be applicable to all `video` elements, but it can not be applicable to all `object` elements used to show video, unless the term "video" is further defined.
+## Accessibility Support
 
-Finding objective definitions to use in rules can be difficult, if not outright impossible in some cases. The intent here is to ensure repeatability of the rule. Not everything in WCAG testing is entirely repeatable, but when it comes to rule applicability, this is a hard requirement.
+Sometimes a feature relevant to a rule is not fully supported by assistive technology and user agents. Include information on any known support issues for a feature relevant to the rule.
 
-For more details, see [ACT Rules Format: Applicability](https://w3c.github.io/wcag-act/act-rules-format.html#applicability)
+For more details, see [ACT Rules Format: Accessibility Support](https://www.w3.org/TR/act-rules-format/#accessibility-support).
 
-## Expectations
+## Background
+
+While optional, this provides background information relevant to the development of the rule. Any document cited in the rule should be included here, along with any applicable success criterion, understanding or technique documents, other specifications, and any other relevant reading. 
+
+For more details, see [ACT Rules Format: Background](https://www.w3.org/TR/act-rules-format/#background).
+
+## Test cases
+
+The test cases are snippets of code that help with understanding and can be used for validating implementations of the rule. There must be at least one example for **pass**, **fail** and **unapplicable** outcomes. 
 
-The applicability help the testers (or test tools) identify what has to be checked. Following that, the expectations are statements that must be true for the applicable elements to pass the rule. If any of the expectations is false, than the target element failed the rule.
+Each test case should be simply and uniquely named, _e.g. "Passed Example 1"_. Each example must also include a brief declarative description that uses plain language, and is a well-formed grammatically correct sentence that finishes with a full stop/period. Words from code should use back ticks and be identified as elements or attributes.
```

Comment:
I would add:
- description should explain why the example has the outcome it claims to have (which is usually not a problem…)
- Use demonstrative form when possible ("this image" vs "the image"); or fully qualified denomination when needed ("the `div` element with an ID of `foo`", if there are several `div` in the example and "this `div`" is not possible; "the first `title` element"; …); or "there is no element with a role of…" (typical for inapplicable examples) (not sure how to name that construction).

Reply:
Suspect we also need some guidance on the code, around ensuring Passed examples would pass all rules, and the relevant thing in a Failed example being the only thing that would not pass all rules.

@WilcoFiers was there anything else in the AG feedback?